tests/main/lxd/task.yaml

```diff
@@ -200,6 +200,20 @@ execute: |
     snap install lxd-demo-server
     snap connect lxd-demo-server:lxd lxd:lxd
 
+    echo "Check that snaps in containers don't prevent refreshes in the host"
+    snap pack "$TESTSLIB/snaps/test-snapd-sh"
+    snap install --dangerous test-snapd-sh_1.0_all.snap
+
+    lxc file push test-snapd-sh_1.0_all.snap my-ubuntu/home/sh.snap
+    lxc exec my-ubuntu -- snap install --dangerous /home/sh.snap
+
+    # keep the snap running in the container until we refresh in the host
```

Comment:
No that shouldn't happen because the container won't show lxc's slice. From inside the container, it will look "normal" (i.e., a system.slice with services and scope in it)